src/design/ButtonBar/index.stories.tsx

```diff
@@ -102,6 +102,35 @@ export const ButtonSizes = () => (
   </div>
 );
 
+export const AsChildButtons = () => (
+  <div className="flex flex-col gap-4">
+    <p>No Click, Hover effects on child components except Anchor</p>
+    <ButtonBar>
+      <Button asChild size="lg">
+        <label>Label1</label>
+      </Button>
+      <Button asChild size="lg">
+        <label>Label2</label>
+      </Button>
+      <Button asChild size="lg">
+        <a href="/">Link</a>
+      </Button>
+      <Button size="lg">
+        <GitBranchIcon /> Review
+      </Button>
+      <Button size="lg">
+        <SearchIcon /> Search
+      </Button>
+      <Button asChild size="lg">
+        <span>Span Tag</span>
+      </Button>
+      <Button variant="primary" asChild size="lg">
+        <span>No Hover</span>
```

Comment:
What does "No Hover" mean here

Reply:
Primary button has the Hover Effect 
But when we apply asChild props, and the child is Span, then there is no hover effect
Only <a> child will have the hover effect, click effect
This is what I wanted to say in the story